PopulateTools/gobierto
# UI for including a page in a section from the edit page screen

Connects to #1016 

### What does this PR do?

- [X] Provides the markup for including a page in a existing section
- [x] Implements it


### How should this be manually tested?

- Add section from page creation
- Update section from page update
- Creation of new section
- Add section without parent
- Add section of a child
- Delete section

After adding the page to the section you can check the tree of the section.

Consideration: When a page belongs to a section several times, only the first section item is shown in the page edition.

### Pending (VUE)

- [x] When you create a new section, the multi-language titles displayed in the window appear. I thought it was relation with turbolinks but I deactivated them and it continue. Also, when you create a section from /admin/cms/pages, the multi-language section options don't appear.
